mne/viz/epochs.py

```diff
@@ -69,6 +71,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    overlay_times : array-like, shape (n_epochs,) | None
+        If not None the parameter is interpreted as time instants in seconds
```

Comment:
okay, I am happy with how it works now. Just one last nitpick @agramfort . I would specify that the `overlay_times` parameter is for the unordered epochs. That is overlay_times[0] corresponds to epochs[0], overlay_times[1] corresponds to epochs[1] etc. So, user should not attempt to order it before passing it to argument ...